clang/include/clang/Driver/Options.td

```diff
@@ -4086,6 +4086,10 @@ def fsycl_exp_range_rounding : Flag<["-"], "fsycl-exp-range-rounding">,
   Visibility<[ClangOption, CLOption, DXCOption, CC1Option]>,
   HelpText<"Use experimental range rounding.">,
   MarshallingInfoFlag<LangOpts<"SYCLExperimentalRangeRounding">>;
+def fsycl_fp64_conv_emu : Flag<["-"], "fsycl-fp64-conv-emu">,
+  Visibility<[ClangOption, CLOption, DXCOption, CC1Option]>,
```

Comment:
Is there a test case to check the visibility of this option in `DXC` mode(`DXCOption`)?

Reply:
Visibility for DXC mode is not necessary here. Removing it from the list. Thanks